DOCUMENTATION.md

```diff
@@ -217,6 +219,153 @@ uint32_t secure_sum(uint32_t op1, uint32_t op2)
     return secure_gateway(my_box_name, __secure_sum, op1, op2)
 }
 ```
+---
+```C
+UVISOR_BOX_NAMESPACE(static const char const namespace[])
```

Comment:
Could you move this part at the end of the **Configuration macros** section? Here it apparently belongs to the **Secure gateway** section.